src/commons/application/ApplicationTypes.ts

```diff
@@ -279,13 +283,15 @@ export const ALL_LANGUAGES: readonly SALanguage[] = [
   fullTSLanguage,
   htmlLanguage,
   ...schemeLanguages,
-  ...pyLanguages
+  ...pyLanguages,
+  ...cLanguages
 ];
 // TODO: Remove this function once logic has been fully migrated
 export const getLanguageConfig = (
   chapter: Chapter,
   variant: Variant = Variant.DEFAULT
 ): SALanguage => {
+  //TODO: come back fix
```

Comment:
Is this TODO meant to be acted on? Or is this PR ready?

Reply:
Seems to be a TODO carried over from when I first branched from frontend. Will remove!